### PR TITLE
Bug/17 zoom stopped working

### PR DESCRIPTION
**Pull request by arunge**
# TraceR 24.09.0

## Bug Fixes
- fix issue with zoom (#17)
- add logo as icon for the browser tab

### CODE REVIEW

File: inst/app/ui.R [hunk 1]
```diff
@@ -37,13 +37,14 @@ tagList(
...
         mainPanel(
-          DiagrammeR::grVizOutput("flowchart", width = "600px", height = "600px"),
+          DiagrammeR::grVizOutput("flowchart", width = "800px", height = "800px"),
           textOutput("clickMessage")
         ),
       )
     )
   ),
   tags$head(
+    tags$link(rel = "icon", type = "image/x-icon", href = "pandora-isomemo.ico"),
```
Comment (arunge) [inline inst/app/ui.R:47]: I added an icon for the browser tab. This removed an error when inspecting the app after loading.

File: inst/app/server.R [hunk 2]
```diff
@@ -39,13 +39,19 @@ shinyServer(function(input, output, session) {
...
+        bounds: true,
+        boundsPadding: 0.1
+      });
+    } else {
+      // Retry after a short delay if the SVG is not available yet
+      setTimeout(initializePanzoom, 100);
+    }
+  }
+  initializePanzoom();  // Initial call to check and start panzoom
+')
```
Comment (arunge) [inline inst/app/server.R:56]: @f-lukas chat-gpt gave me a more robust that checks if "#flowchart svg" already exists before loading panzoom.

**Comment by arunge** [comment]
@f-lukas I found how to fix the zoom. Please check the changes and feel free to merge! :pray:

File: inst/app/server.R (hunk 2 above)
Comment (f-lukas) [reply inst/app/server.R:56]: So simple and a great idea. Thanks a lot for looking into this.

**Review by f-lukas — APPROVED** [review]

**Pull request merged**